yammer-metrics/src/test/java/com/wavefront/integrations/metrics/WavefrontYammerMetricsReporterTest.java

```diff
@@ -190,27 +206,27 @@ public void testPlainTimer() throws Exception {
     timer.time().stop();
     wavefrontYammerMetricsReporter.run();
     assertThat(receiveFromSocket(15, fromMetrics), containsInAnyOrder(
-        startsWith("\"mytimer.count\""),
-        startsWith("\"mytimer.min\""),
-        startsWith("\"mytimer.max\""),
-        startsWith("\"mytimer.mean\""),
-        startsWith("\"mytimer.sum\""),
-        startsWith("\"mytimer.stddev\""),
-        startsWith("\"mytimer.median\""),
-        startsWith("\"mytimer.p75\""),
-        startsWith("\"mytimer.p95\""),
-        startsWith("\"mytimer.p99\""),
-        startsWith("\"mytimer.p999\""),
-        startsWith("\"mytimer.m1\""),
-        startsWith("\"mytimer.m5\""),
-        startsWith("\"mytimer.m15\""),
-        startsWith("\"mytimer.mean\"")
+        startsWith("\"mytimer.duration.min\""),
```

Comment:
hm, we may need to convey this change to users or, alternatively, we may need to stick to the old behavior